src/views/studio/modals/transfer-host-selection.jsx

```diff
@@ -56,7 +56,7 @@ const TransferHostSelection = ({
                                 className={classNames('button', {
                                     'mod-mutating': loading
                                 })}
-                                onClick={onLoadMore}
+                                onClick={() => onLoadMore()} // eslint-disable-line react/jsx-no-bind
```

Comment:
It might be good to make this change when we're defining `onLoadMore` instead (e.g. line 114 of this file)

Reply:
nice! fixed.